.ci/docker-compose.test.yml

```diff
@@ -0,0 +1,21 @@
+version: '3.8'
+
+services:
+  test:
+    image: ${TEST_IMAGE_TAG:-test}
+    depends_on: 
+      - database
+    environment:
+      PARALLEL_WORKERS: 1 # makes tests run sequentially instead of simultaneously
+    command: >
+      bash -c "bin/rake spec"
+      
+  database:
+    image: ${DB_IMAGE_TAG:-db}
+    restart: unless-stopped
+    ports: 

```

Comment:
It doesn't matter IMO here, bu just a reminder we can always use networks instead of exposing ports